MainWindow.xaml.cs

```diff
@@ -0,0 +1,2329 @@
+            if (pierwszaLinia != null)
+            {
+                liczbaKolumn = pierwszaLinia.Split(delimiter).Length;
+            }
+            string[,] tablicaString4 = new string[liczbaLinii, liczbaKolumn];
+            double[,] tablicaDouble4 = new double[liczbaLinii, liczbaKolumn];
+            tablicaString4 = DoTablicyString(filename4, liczbaLinii, liczbaKolumn, delimiter);
+            tablicaDouble4 = DoTablicyDouble(tablicaString4, liczbaLinii, liczbaKolumn, filename4);
+            for (int q = 0; q < liczbaKolumn; q++)
+            {
+                pomocMax1.Add(ZnajdzMinMax(tablicaDouble4, liczbaLinii, liczbaKolumn)[1, q]);
+            }
+            List<string> typyDanychPomoc4 = new List<string>();
+            typyDanychPomoc4 = SprawdzTypDanych(filename4, delimiter, liczbaLinii, liczbaKolumn);
+            for (int q = 0; q < liczbaKolumn; q++)
+            {
+                pomocMin1.Add(ZnajdzMinMax(tablicaDouble4, liczbaLinii, liczbaKolumn)[0, q]);
+            }
+            Configuration config4 = new Configuration()
+            {
+                Separator = ",",
+                LiczbaLinii = File.ReadLines(filename4).Count(),
+                LiczbaKolumn = File.ReadLines(filename4).First().Split(delimiter).Length,
+                ZakresAtrybutow = new List<List<string>>()
+                {
+                    new List<string>(){$"{pomocMin1[0]}", $"{pomocMax1[0]}"},
+                    new List<string>(){$"{pomocMin1[1]}", $"{pomocMax1[1]}"},
+                    new List<string>(){$"{pomocMin1[2]}", $"{pomocMax1[2]}"},
+                    new List<string>(){$"{pomocMin1[3]}", $"{pomocMax1[3]}"},
+                    new List<string>(){"Iris-setosa", "Iris-versicolor", "Iris-virginica"}
+                },
+                WartosciMinimalneWKolumnie = pomocMin4,
+                WartosciMaksymalneWKolumnie = pomocMax4,
+                TypyDanych = typyDanychPomoc4,
+                ZnakZapytania = "brak",
+                Decyzja = liczbaKolumn,
+            };
+
+
+            string config1Json = JsonConvert.SerializeObject(config1);
+            string config2Json = JsonConvert.SerializeObject(config2);
+            string config3Json = JsonConvert.SerializeObject(config3);
+            string config4Json = JsonConvert.SerializeObject(config4);
+
+            SaveFileDialog saveFile = new SaveFileDialog();
+            saveFile.Filter = "Json File|*.json";
+            saveFile.Title = "Zapisz config";
+
+            bool? result = saveFile.ShowDialog();
+            if (result == true)
+            {
+                StreamWriter configJson = new StreamWriter(saveFile.FileName);
+                configJson.Write(config1Json);
+                configJson.WriteLine("\n" + config2Json);
+                configJson.WriteLine(config3Json);
+                configJson.WriteLine(config4Json);
+                configJson.Flush();
+                configJson.Close();
+            }
+        }
```

Comment:
troche srednia ta funkcja bo masz wszystko na stale powpisywane, nic nie ma dynamicznie mozna powiedziec. To tak jakbys mogl w ogole wywalic tworzenie configu i bedziesz mial mniej linijek

Reply:
a nie bylo wymagane przeciez tworzenie configu.